virus/Hemileia_pugnator/Hemileia pugnator.md

```diff
@@ -0,0 +1,132 @@
+
+
+![Ecology](Table1_Bioprojects_Samples_Runs_Hemileia_pugnator)
+
+The common denominator is the presence of Hemileia vastatrix race XXXIII. Therefore, in accordance with the observation that mitoviruses are preferentially located in the mitochondria of plant pathogenic fungi, and considering the report of mitoviruses in a basidiomycota causing a rust-like disease [Liu et al. 2016], Hemileia pugnator is postulated as a new mitovirus infecting the mitochondria of Hemileia vastatrix race XXXIII. In the case of transcriptomic analysis of non-infected coffee plants, it is conceivable that the genome of Hemileia pugnator was integrated into the mitochondrial genome of these plants during an ancestral infection by Hemileia vastatrix race XXXIII or an unknown mitovirus sharing a high identity degree. Due to the intracellular nature of the infection, the fusion between Hemileia vastatrix cells and the coffee plant cells might have resulted in the integration of the mitovirus into the plant mitochondria as a cryptic virus, which could be present in the descendants of these coffee plants. However, the mechanisms underlying this phenomenon, observed between various mitoviruses and plants [Nibert et al. 2018], remain undefined.
+
+### Virus Genome {Q3}
+
+*Hemileia pugnator* genome with key features can be visualized in the `H_pugnator_genome.png` file. All the sequences related to *Hemileia pugnator* (genome, RdRp) can be found in the text file `H_pugnator_seq`. 
+
+Its genome is composed of one single open reading frame,as assessed by the ORF Finder tool available on NCBI, in the positive sens of the RNA which corresponds to the characteristics of mitoviruses. Moreover, this single open reading frame is predicted to encode for a protein with a mitovirus-like RdRp functional domain using InterProScan (cf `H_pugnator_fct_domains.xlm`).
+
+[Hemileia pugnator ORF](https://www.ncbi.nlm.nih.gov/projects/sviewer/?id=lcl|1&tracks=[key:sequence_track,name:Sequence,display_name:Sequence,id:STD649220238,annots:Sequence,ShowLabel:false,ColorGaps:false,shown:true,order:1][key:gene_model_track,name:UB9opAA2mDBA4,display_name:(U)ORFfinder_11.23.15168173,id:UB9opAA2mDBA4,data_key:iTYf5-k0PhUSGggSOQs2HGVhZ2BvYkNoS3BFWtBX5IKMtFqbcXZA7waQB-lS9QvtGcVOxA3aHcsH0RfOIeMY4TjRHg,dbname:NetCache,annots:ORFfinder_11.23.15168173_UUD1700770569DUU_gene_model,Options:MergeAll,CDSProductFeats:false,NtRuler:true,AaRuler:true,HighlightMode:2,ShowLabel:true,highlights_color:#0000ff,shown:true,order:9]&key=GKeOdnilr4SDi5mDqJqnjfTw9vH-89L52uHUy0HGdRMdJcRKrVecPVUCBHtRZwh_GldNVg5IHlkEQxRcInEbcztDHQ&mk=158:2269|ORF4|0000ff|9&noguess_assm=1&v=1:2435&c=null&select=null&slim=0&toolbar=-d)
+
+![Hemileia pugnator genome map](H_pugnator_Genome.PNG)
```

Comment:
The `.png` file link for these should use the path of the file, right now this will not render on the MD page since it's pointing to a `.png` in exactly this directory.